src/components/Auth.jsx

```diff
@@ -65,11 +65,12 @@ const Auth = () => {
           )}
           {user && (
             <>
-              <NavLink to={`/profile/${auth?.currentUser?.uid}`} className="hidden md:block text-black bg-gray-400 hover:bg-gray-500 rounded-md px-3 py-2">
-                <div className="flex">
-                  Welcome {user.displayName}
-                  {unreadMessages && <RiErrorWarningFill className="l-1 w-5 h-5"/>}
-                </div>
+              <NavLink to={`/profile/${auth?.currentUser?.uid}`} className="hidden md:block">
+                <img
+                  src={user.photoURL} // User's profile picture
```

Comment:
I would prefer the original version
Reason: profile picture might not load in some cases and it will look bad